contents/product-engineers/best-customer-retention-metrics.md

```diff
@@ -0,0 +1,246 @@
+* **Passives**, who gave a score of 7 or 8, aren’t super happy or unhappy but are also unlikely to promote your business. 
+* **Detractors**, those who rated you between 0 and 6, could potentially damage your business through word-of-mouth.
+
+As you might imagine from a metric that relies on people answering a single question, the NPS is both alluring and [shaky](https://www.wsj.com/articles/the-dubious-management-fad-sweeping-corporate-america-11557932084). Alluring because, amongst a sea of often confusing data points it can seemingly provide clarity. But shaky because, well, a single data point isn’t enough to go on. As a result, it’s important to combine it with other metrics, including user surveys and interviews, to really understand what’s going on.
+
+### How is it useful?
+
+Pretty much all subscription-based businesses should be measuring their NPS as customer satisfaction is a key driver of churn. This is especially true for e-commerce businesses – if your online store is poorly designed and hard to use, your chances of repeat business will plummet. 
+
+![NPS survey](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/nps-csat-ces/nps-survey.png)
+
+This isn’t the most useful metric for very early-stage startups. NPS relies on having a large number of established customers to provide meaningful data. CSAT and CES scores, however, are very useful metrics for early-stage startups. More on those a little lower down.
+
+### How to calculate your NPS 
+
+The NPS score is calculated by subtracting the percentage of Detractors from the percentage of Promoters. For example, if 80% of your customers were promoters and 5% detractors, your NPS score would be 75. Anything above 0 is good, above 20 is great, and above 50 is very good.
+
+### What’s a good NPS?
+
+According to [Satmetrix](https://www.satmetrix.com/wp-content/uploads/2023/07/NICE-Net-Promoter-Benchmarks-2023.pdf), the average NPS score for software and apps is 27. But there’s a really wide range – at Dropbox, it’s 27, at Slack it’s 53, and at Square it’s 70. At PostHog, the NPS for our product analytics is 46.7, for our session replay it’s 56.6, and for feature flags it’s 65.9.
+
+## 5. Customer satisfaction score (CSAT)
+
+### What is a CSAT?
+
+Your customer satisfaction score, or CSAT, is similar to your NPS, but is specifically a great customer retention metric to use before you’ve found product-market fit. Satisfaction tends to be short-lived, whereas loyalty takes time to build. And that’s why CSAT is a great metric for early-stage startups.
+
+### How is it useful?
+
+Any businesses that interact in a very direct, tangible way with customers should be measuring their CSAT: think retailers, restaurants, and customer support. On the flipside, businesses that rely on very transactional relationships with customers (gas stations) or high-value, single-purchase businesses (car dealerships) should place higher importance on other metrics.
```

Comment:
As per earlier comments, we should avoid examples about "non tech" businesses as they're not really relevant to our audience.